packages/utils/src/schema/getDefaultFormState.ts

```diff
@@ -272,7 +280,7 @@ export default function getDefaultFormState<
   theSchema: S,
   formData?: T,
   rootSchema?: S,
-  includeUndefinedValues = false
+  includeUndefinedValues: boolean | "once" = false
```

Comment:
Also, update the function docs to add details for the new option
```suggestion
  includeUndefinedValues: boolean | "noRecursion" = false
```